packages/block-library/src/post-featured-image/edit.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * WordPress dependencies
+ */
+import { useEntityProp, useEntityId } from '@wordpress/core-data';
+import { useSelect } from '@wordpress/data';
+import { ResponsiveWrapper } from '@wordpress/components';
+
+function PostFeaturedImageDisplay() {
+	const [ featuredImage ] = useEntityProp(
+		'postType',
+		'post',
+		'featured_media'
+	);
+	const media = useSelect(
+		( select ) =>
+			featuredImage && select( 'core' ).getMedia( featuredImage ),
+		[ featuredImage ]
+	);
+	return media ? (
+		<ResponsiveWrapper
+			naturalWidth={ media.media_details.width }
+			naturalHeight={ media.media_details.height }
+		>
+			<img src={ media.source_url } alt="Post Featured Media" />
```

Comment:
This tag should have a class name so that it can be targeted by Theme and Plugin developers.